test/unit/ui/camera.test.js

```diff
@@ -316,6 +316,34 @@ test('camera', (t) => {
         t.end();
     });
 
+    t.test('#setPadding', (t) => {
+        t.test('sets padding', (t) => {
+            const camera = createCamera();
+            const padding = { left: 300, top: 100, right: 50, bottom: 10};
+            camera.setPadding(padding);
+            t.deepEqual(camera.getPadding(), padding);
+            t.end();
+        });
+
+        t.test('doesnt change padding thats already present if new value isnt passed in', (t) => {
```

Comment:
test that padding is retained when no padding is set to camera might be useful, too

Reply:
Added 👍